assets/scss/base/_lists.scss

```diff
@@ -63,6 +94,23 @@ ul, nav ul {
 	}
 }
 
+/*
+Activity list
+
+An definiton list where the list uses the class `.activity-list` and the child elements uses the element class `.activity-list__date` and it's siblings use `.activity-list__description`
+
+Markup:
+<dl class="activity-list">
+  <dt class="activity-list__date">14th March 2016</dt>
+  <dd class="activity-list__description">description</dd>
+  <dd class="activity-list__description">description</dd>
+  <dd class="activity-list__description">description</dd>
+  <dd class="activity-list__description">description</dd>
+</dl>
+
+Styleguide Lists.activity list
+*/
+
 .activity-list {
 	padding-left: 0;
 	li {
```

Comment:
@rpowis date and description sound like a dl to me. Tell me if you think otherwise.